client/handler_test.go

```diff
@@ -328,6 +328,15 @@ func TestHandler(t *testing.T) {
 					path:       client.ClientsHandlerPath,
 					statusCode: http.StatusBadRequest,
 				},
+				{
+					d: "setting skip_consent fails",
+					payload: &client.Client{
+						RedirectURIs: []string{"http://localhost:3000/cb"},
+						SkipConsent:  true,
+					},
+					path:       client.DynClientsHandlerPath,
+					statusCode: http.StatusBadRequest,
```

Comment:
We test that we can't set it, but how do we test that we can set it?

Reply:
In the E2E tests setting is tested.

Reply:
Here: https://github.com/ory/hydra/pull/3451/files#diff-62406830adc9cbc1fec4e4768b0debd0379361f10a6786ce4b933390bcf5523dR96

Reply:
Maybe one explicit test? :)

Reply:
done.